youtube_dl/extractor/vtv.py

```diff
@@ -0,0 +1,91 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+import re
+
+from ..utils import extract_attributes
+
+class VTVIE(InfoExtractor):
+    _VALID_URL = r'https?://(au|ca|cz|de|jp|kr|tw|us|vn)\.tvnet\.gov\.vn/[^/]*/(?P<id>[0-9]+)/?'
```

Comment:
Don't capture groups you don't use. Use proper regex to match all country codes.

Reply:
These are the only country codes that are valid at the moment. Do you think it'd be better to expand it to capture every country code, or even [a-z][a-z]?